examples/tree_3d_dgsem/elixir_mhd_dynamo.jl

```diff
@@ -0,0 +1,167 @@
+# A turbulent helically driven dynamo model in resistive compressible MHD.
+# We start with a weak small-scale magnetic field provided as Gaussian noise.
+# The energy is supplied via a forcing function f_force(x, t), which is delta-correlated in time.
+# For a comparison look into the PC run pencil-code/simon/forced/alpha2_periodic/64_kf_5_a.
+
+using OrdinaryDiffEq
+using Trixi
+using Random
+using LinearAlgebra
+
+###############################################################################
+# initial condition and forcing function
+
+"""
+    initial_condition_gaussian_noise_mhd(x, t, equations)
+
+Define the initial condition with a weak small-scale Gaussian noise magnetic field.
+"""
+function initial_condition_gaussian_noise_mhd(x, t, equations)
+    amplitude = 1e-8
+
+    seed = reinterpret(UInt64, t + x[1] + x[2] + x[3])
```

Comment:
Changed it to target unsigned integer type `UInt`. Since this defines the seed of a random number the results are not binary reproducible on different systems.